app/controllers/admin/loans_controller.rb

```diff
@@ -191,10 +191,18 @@ def prep_transactions
       @transactions = ::Accounting::Transaction.where(project: @loan).extracted
       @transactions.includes(:account, :project, :currency, :line_items).standard_order
       @enable_export_to_csv = true
-      @transactions_grid = initialize_grid(@transactions, order: 'accounting_transactions.txn_date',
-                                                          enable_export_to_csv: @enable_export_to_csv,
-                                                          per_page: 100, name: 'transactions')
-      export_grid_if_requested('transactions': 'admin/accounting/transactions/transactions_grid_definition')
+      @transactions_grid = initialize_grid(
+        @transactions,
+        order: "accounting_transactions.txn_date",
+        custom_order: {
+          "accounting_transactions.txn_date" =>
+            "accounting_transactions.txn_date, length(accounting_transactions.loan_transaction_type_value)"
+        },
```

Comment:
example wicegrid custom ordering here - the order value gets overriden by valid sql for ORDER_BY.  https://github.com/leikind/wice_grid#custom-ordering